rapyer/scripts/lua/sf/redis_priority_queue/save.lua

```
@@ -0,0 +1,4 @@
+function(special_key, payload)
```

Luacheck (1.2.0)

[error] expected identifier near '('

(E011)